src/app/shared-services/db/sqlite.service.ts

```diff
@@ -333,6 +334,74 @@ export class SqliteService {
     )
     return durationDays
   }
+
+  async getStaypoints(id: string): Promise<StayPoints> {
```

Comment:
We might want to call this parameter `trajectoryId` to make it clear what kind of ID is meant.